matchbox/resource_group_test.go

```diff
@@ -0,0 +1,65 @@
+package matchbox
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/coreos/matchbox/matchbox/storage/testfakes"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestResourceGroup(t *testing.T) {
+	srv := NewMatchboxFixtureServer(testfakes.NewFixedStore())
```

Comment:
In my first version, I was calling Stop, but looks like the server has some issues:

```
2017/05/23 23:46:49 transport: http2Client.notifyError got notified that the client transport was broken EOF.
2017/05/23 23:46:49 transport: http2Server.HandleStreams failed to read frame: read tcp 127.0.0.1:33703->127.0.0.1:42876: use of closed network connection

```

Since is not doing a clean stop.